Assets/Scripts/API/Save/SaveTreeHeader.cs

```diff
@@ -64,25 +66,20 @@ void ReadRawData()
             MemoryStream stream = new MemoryStream(RawData);
             BinaryReader reader = new BinaryReader(stream, Encoding.UTF8);
 
-            // Read Version - must be 0x26
-            Version = reader.ReadByte();
-            if (Version != VersionNumber)
-                throw new Exception("SaveTree file has an invalid version number, must be 0x26.");
-
-            // Read CharacterPosition.RecordType - must be 0x01
-            CharacterPosition = new HeaderCharacterPositionRecord();
-            CharacterPosition.RecordType = reader.ReadByte();
-            if (CharacterPosition.RecordType != (int)RecordTypes.CharacterPosition)
-                throw new Exception("Expected CharacterPosition in SaveTreeHeader has an invalid record type, must be 0x01.");
-
-            // Read CharacterPosition.Unknown
-            CharacterPosition.Unknown = reader.ReadUInt16();
+            // Read Version - must be 0x125 or 0x126
+            Version = reader.ReadInt32();
+            if (Version < VersionNumberOld || Version > VersionNumberNew)
```

Comment:
I found a few on my first release haha. There's definitely people out there who never patched. :) That's why I put in the check, as these saves don't always load properly (slightly different format).

Classic might accept 1.0, but it has the luxury of 100% knowing the file format. We're kind of piecing it together and it's enough just to support .213 I think.